core/src/main/java/org/axonframework/eventsourcing/eventstore/EventStorageEngine.java

```diff
@@ -113,13 +113,12 @@ default DomainEventStream readEvents(String aggregateIdentifier) {
     DomainEventStream readEvents(String aggregateIdentifier, long firstSequenceNumber);
 
     /**
-     * Try to load a snapshot event of the aggregate with given {@code aggregateIdentifier}. If the storage engine has
-     * no snapshot event of the aggregate, an empty Optional is returned.
+     * Try to load a snapshot events of the aggregate with given {@code aggregateIdentifier}.
```

Comment:
Typo: I'd remove `a` in this sentence, so:
`Try to load snapshot events of the aggregate with given {@code aggregateIdentifier}.`